src/lib/services/user-service.ts

```diff
@@ -136,28 +136,32 @@ class UserService {
     ): Promise<void> {
         let username: string;
         let password: string;
+        let email: string;
 
         if (
             initialAdminUserConfig.createAdminUser !== false &&
             initialAdminUserConfig.initialAdminUser
         ) {
             username = initialAdminUserConfig.initialAdminUser.username;
             password = initialAdminUserConfig.initialAdminUser.password;
+            email = initialAdminUserConfig.initialAdminUser.email;
         } else {
             username = 'admin';
             password = 'unleash4all';
+            email = 'admin@unleash.com';
         }
 
         const userCount = await this.store.count();
 
-        if (userCount === 0 && username && password) {
+        if (userCount === 0 && username && password && email) {
             // create default admin user
             try {
                 this.logger.info(
-                    `Creating default user '${username}' with password '${password}'`,
+                    `Creating default user '${username}', password '${password}', and email '${email}'`,
```

Comment:
I think we can leave the old "with" in here, personally. Not major, but it keeps the format slightly more consistent. Not that it matters much 🤷🏼 

```suggestion
                    `Creating default user '${username}' with password '${password}' and email '${email}'`,
```